### ISSUE TITLE
Communication

### ISSUE THREAD

**Issue by mqudah91** [ArabicMozilla/Planning#4]
## Objective Goal:
By the end of December 2016, X% of the community is documenting their activities in github and they are reported in the monthly meeting ,if we can collect up to 3 reports that will be good because the maximum number is 7.
## Roles
* **R**esponsible: @mqudah91
* **A**ccountable: @mqudah91
* **S**upporter: @anefzaoui 
* **C**onsulted: @bacharakis @ppapadeas @nukeador 
* **I**nformed: @ppapadeas  @bacharakis @nukeador 
## Item list:
- Actual video calls (R: @mqudah91).
- Reporting (Report Structure ), each community has to submit reports for their activities to (R: @ZaafouriFF  @mqudah91)
- Mozilla should assign someone to follow up the community (R: @nukeador )
- Make sure that Arabic community sync with external community (R: @ZaafouriFF  @mqudah91)

##Activity:
1- bi-weekly call or monthly.
2- in the call the note taker will write the report.
3- the chair for the call who will be the same guy who assigned by Mozilla.
4- the following up who is the responsible to share the new news with the Arabic community in Arabic to make sure they understand all of the idea or the topic .
5- take a place for community in global events (like : FOSDOM , All Hands , Summits )

**Comment by nukeador** [comment]
Feedback: What's the concrete goal for this task? It should be Specific, Measurable, Achievable, Relevant, Time-bound.

Example:

> By the end of December 2016, the community is meeting on monthly basis with at least X% of the community attending or providing comments.

And maybe another different issue should be:

> By the end of December 2016, X% of the community is documenting their activities in github and they are reported in the monthly meeting.

**Comment by deimidis** [comment]
Hi @mqudah91 did you had the chance to work on this issue? There's a feedback from @nukeador too that is important to try to measure if you succeed or not :)

**Comment by mqudah91** [comment]
Yes am already working on that ... am asking communities to publish report on etherpads or her , thats to make them visible .
And am trying to arrange monthly call if there is somthing important to talk on at least with responsible people in communities.

**Comment by deimidis** [comment]
Great @mqudah91 please read @nukeador comments and try to update the issue reflecting changes on the goal and how we will see if we succeed or not :)

**Comment by mqudah91** [comment]
@deimidis like this ?
